### PR TITLE
Add rerun.io to the op.sh tool

### PR DESCRIPTION
**Pull request by pfeiferj**
**Description**

Adds rerun as a sub-command in the op.sh tool. Also updates the help text to suggest trying the rerun demo instead of the plotjuggler demo as plotjuggler no longer works inside of the devcontainer.

**Verification**

Ran the command on my computer.

### CODE REVIEW

File: tools/op.sh [hunk 1]
```diff
@@ -331,6 +336,7 @@ function op_default() {
   echo ""
   echo -e "${BOLD}${UNDERLINE}Commands [Tooling]:${NC}"
   echo -e "  ${BOLD}juggle${NC}       Run PlotJuggler"
+  echo -e "  ${BOLD}rerun${NC}        Run rerun"
```
Comment (maxime-desroches) [inline tools/op.sh:339]:
```suggestion
  echo -e "  ${BOLD}rerun${NC}        Run Rerun"
```

File: tools/op.sh [hunk 2]
```diff
@@ -356,8 +362,8 @@ function op_default() {
   echo "  op build -j4"
   echo "          Compile openpilot using 4 cores"
   echo ""
-  echo "  op juggle --demo"
```
Comment (maxime-desroches) [inline tools/op.sh:359]: Let's keep the old juggle example. Rerun is still a work in progress.

**Comment by adeebshihadeh** [comment]
Rerun is still very much in development, so I don't think we want to add to `op` yet.

Are you already using it and prefer it over PJ?

**Comment by pfeiferj** [comment]
@adeebshihadeh I've started using it because plotjuggler doesn't work in the devcontainer due to the capnp dependency. I wouldn't necessarily say I prefer it, haven't done enough "advanced" stuff with it to really say. I do miss having the option of using the qlog files though.

**Comment by maxime-desroches** [comment]
Going to close this. Rerun is not quite ready (yet) to offer a good development experience

**Pull request closed without merging**